vertica_python/vertica/connection.py

```diff
@@ -151,10 +151,10 @@ def write(self, message):
 
         except Exception as e:
             self.close_socket()
-            if e.message == 'unsupported authentication method: 9':
+            if str(e) == 'unsupported authentication method: 9':
```

Comment:
I'm not sure this holds. But we can no longer use `message` as that attribute has been removed from `BaseException`.